src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -118,58 +118,58 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AvoidAsyncVoidTitle" xml:space="preserve">
-    <value>Avoid Async Void</value>
+    <value>Avoid 'async void'.</value>
```

Comment:
💡 This needs `{Locked="async void"}` as the message. The same would apply for other uses of keywords and literals.

Reply:
Is it a roslyn/roslyn-analyzer specific syntax?

Reply:
@Evangelink This is used by the localization team to tell them don't translate this part. Probably in any project in the organization.

@sharwell Trying to apply Locked everywhere in the PR will make it VERY big, and possibly hard to review. But let me know if you still prefer to do that.